src/components/toast/toast.stories.mdx

```diff
@@ -8,19 +8,23 @@ import Button from '../button';
 <Meta title="Design System/Toast" parameters={{ info: { disable: true }, chromatic: { disable: true }}} />
 
 # Toast
+Toast is part of Carbon's Flashes components.
+Its purpose is to present quick confirmation of success or failure, relating to the task the user has just performed. Alternatively, use Toast messages for a longer, timely message that the user must dismiss.
 
 ## Quick Start
 
 To use toast, import the `Toast` and pass the content as children.
 
+```jsx
+import Toast from 'carbon-react/lib/components/toast';
+```
+## Developer Notes
 It can be used as a controlled component where the open state is controlled externally,
 in that case both `onDismiss` and `open` props are required.
 
```

Comment:
lack of `Examples` section

Reply:
Just looked Examples section is below the developer notes